src/webgpu/web_platform/copyToTexture/video.spec.ts

```diff
@@ -11,5 +11,87 @@ TODO: plan
 
```

Comment:
I think both TODOs above can be removed now.

Reply:
I think both TODOs could be removed